cli/roles.go

```diff
@@ -1,27 +0,0 @@
-package main
```

Comment:
This was removed but this can actually be reinstated _and_ implemented.

Because our new flow doesn't require an MFA attempt every single time we retrieve a SAML assertion, we could actually retrieve a SAML assertion and interrogate it for the roles for a given account and then discard the SAML assertion afterwards